app/helpers/application_helper.rb

```diff
@@ -991,6 +991,12 @@ def controller_for_stack(model)
 
   def vm_model_from_active_tree(tree)
     case tree
+    when :automation_manager_cs_filter_tree
```

Comment:
@hstastna `vm_model_from_active_tree` method is to get model for VM explorer trees, let's add changes in this PR to a new method specific to Configuration manager records.